sherpa/astro/tests/test_astro_data_swift_unit.py

```diff
@@ -431,8 +454,20 @@ def test_can_use_swift_data(make_data_path):
     # this range is used. Alternatively, channel 801 could have been
     # excluded explicitly.
     #
+    # With changes to PHA filtering we now have to change the
+    # filter range again to include the 0.29-0.30 bin.
+    #
     # ui.notice(0.3, 8.0)
-    ui.notice(0.3, 7.995)
+    # ui.notice(0.3, 7.995)
+    # ui.notice(0.299, 8.0)
```

Comment:
Do we need this extra commented line?

Reply:
These were added as I struggled to select the same range of channels as the code changed (it turns out we do things differently to XSPEC ...) and these comments are the scars to prove it. I can remove them.

Reply:
I've cleaned up that comment (more than just removing the unnecessary lines) and noted that we didn't actually need to change the filter command (we may in one of the later PRs - I spent a  long time moving commits around to get a meaningful set of PRs and maybe that change sneaked through, or maybe it was due to some work that I then changed - so that will be interesting to check in those PRs, but not relevant here).